### DISCUSSION TITLE
v2.1.8

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#3248] [分享 / Share]
✨**正式版** `v2.1.8`

<details>
<summary>获得 v2.1.7 预览版的功能</summary>

- `倍速增强` 正式完成分离, 变更为 `记忆倍速`, `扩展倍速` 和独立的插件 `视频倍速 - 快捷键支持`, 详见 [PR 说明](https://github.com/the1812/Bilibili-Evolved/pull/2746). (PR #2746 by [JLoeve](https://github.com/LonelySteve))
- 为通用设置增加了一些选项描述. (#3140)

</details>

🐛修复
- 修复在线仓库把 Toast 消息挡住了. (#2613)
- 修复 `清爽首页` 中番剧时间表不能点击的问题. (#3166)
- 新版动态相关修复: (#3191)
  - 恢复支持的功能: 快速收起评论区, 复制动态链接, 导出动态图片, 展开动态内容, 禁止跳转动态详情, 动态过滤器, 直播信息扩充.
  - `自定义顶栏` 的位置已适配.
  - 还未恢复的有: 夜间模式, 动态反折叠.
> 虽然这次新版动态导致挂了很多功能, 不过技术实现上比旧版好了不少, 官方总算是有点进步了

- 修复 `控制栏触摸优化` 和 `启用双击控制` 在视频页的样式适配.
> 我实在不能理解, 为啥视频页控制栏改得这么宽, 番剧区又很窄...

----

✨**预览版** `v2.1.8-preview`

- 新增功能 `返回原版直播间`. (#2133)
- `外置稍后再看` 和 `启用快速收藏` 支持在稍后再看/收藏夹播放页面显示的选项. (#3138)
- `夜间模式` 开启时能够将 Safari 主题颜色也改成黑色了. (#2384)
- `BV 号转换` 支持复制时带上标题. (#2557)
- Esc 键可以退出 `查看封面` 的图片查看器了. (#2574)
- 操作 `在线仓库` 时, 设置面板将保持打开. (#2630)
- 操作 `自定义顶栏` 的设置时, 设置面板将自动关闭, 避免和 `v1 风格设置面板` 一起使用时顶栏被挡住. (#2642)

🐛修复
- 修复更新组件失败时的报错信息不准确.
- 删除了没有用的功能: 直播画中画, 直播间自动抽奖.

☕开发者相关
- 修复 `addData` 在数据未加载过时就执行 `provider`, 改善文档. (PR #3160 by [timongh](https://github.com/timongh))
- 添加了 `pascalCase` 工具函数.
- 接入 define API (兼容模式, 未知类型默认 any), 新功能都将遵循这套 API 来编写. (PR #3181, #3041 by [timongh](https://github.com/timongh))
- 整理了一下 ESLint 配置, task 不再输出到 HTML 文件, 并移除 `settings.json` 中冗余的配置. (#3202)
- 改进了功能代码加载的稳定性. (PR #3195 by [timongh](https://github.com/timongh))

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.8'>v2.1.8</a>.</em>

**Comment by kethoven** [comment]
预览版第2条，好像没变啊

**Reply by the1812** [reply]
更新组件后, 去组件选项里打开, 默认是关闭的

**Reply by kethoven** [reply]
更新了组件，也换过浏览器尝试，好像都没有变，没有更新

**Reply by the1812** [reply]
commit 是多少, 2f6e86289 才有

**Reply by kethoven** [reply]
Commit: 84b92e6e4

**Reply by the1812** [reply]
更新源检查下, 目前没到 24h 应该只有 GitHub 能更新到最新

**Reply by kethoven** [reply]
> 更新源检查下, 目前没到 24h 应该只有 GitHub 能更新到最新

好的，我等过段时间再看

**Reply by kethoven** [reply]
> 更新源检查下, 目前没到 24h 应该只有 GitHub 能更新到最新

更新了。这个新功能，很好用，很方便。

**Reply by xiaohuangsu** [reply]
传统连播模式没用啊，到多p视频也不会自动切换，只会记住原来是开启连播还是关闭连播
